src/__experimental__/components/decimal/decimal.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  number, select
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import Decimal from './decimal.component';
+import Textbox from '../textbox';
+import getTextboxStoryProps from '../textbox/textbox.stories';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import notes from './documentation';
+
+const store = new Store({
+  value: Decimal.defaultProps.value
+});
+
+const setValue = (evt) => {
```

Comment:
`ev` instead of `evt`
https://github.com/Sage/carbon/wiki/Codebase-Conventions#action-argument-names